### ISSUE TITLE
Extend the public link with an option to autostart Diashow

### ISSUE THREAD

**Issue by michael-kaufmann** [pulsejet/memories#1351]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
My Idea is, to display albums on a raspberry pi running a chromium browser in kiosk mode.
The Raspberry is used as a digital photo frame and nextloud is NOT installed on that machine.

The chromium browser will open a public link and diashow starts automatically without user intervention.

Almost everything is already working well, the only small part which is missing is the autostart of the diashow.
Ofcourse it would be fantastic if the diashow would be looping.

To summarize: Extend the public link with an option to autostart Diashow (and maybe endless loop)

**Describe alternatives you've considered**
Alternative might be to install Nextloud client on Raspberry, Sync the Photos which should be presented and run a local slideshow app.
This would mean many negative aspects, images would be copied, local space required, modification of Slideshows are much more
complicated compared to create new Album in Memories.

**Additional context**
no

**Comment by Shadowfita** [comment]
Yes please! This would be amazing.

**Comment by DonQuichotte76** [comment]
This feature with a random function to show pictures would be very usefull for Diashows on tv e.g.
